Packages/Testing-MIES/UTF_BasicHardwareTests.ipf

```diff
@@ -2905,6 +2905,118 @@ Function TPDuringDAQWithoodDAQ_REENTRY([str])
 	CHECK_EQUAL_TEXTWAVES(stimsets, {"TestPulse", "StimulusSetC_DA_0", "StimulusSetC_DA_0", "", "", "", "", "", ""}, mode = WAVE_DATA)
 End
 
+Function TPDuringDAQTPStoreCheck_IGNORE(device)
+	string device
+
+	PGC_SetAndActivateControl(device, "check_Settings_RequireAmpConn", val = 0)
+
+	PGC_SetAndActivateControl(device, "check_Settings_TP_SaveTP", val = 1)
+
+	PGC_SetAndActivateControl(device, GetPanelControl(0, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(0, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "TestPulse")
+
+	PGC_SetAndActivateControl(device, GetPanelControl(1, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(1, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "PulseTrain_10Hz_DA_0")
+
+	PGC_SetAndActivateControl(device, GetPanelControl(2, CHANNEL_TYPE_HEADSTAGE, CHANNEL_CONTROL_CHECK), val = 1)
+	PGC_SetAndActivateControl(device, GetPanelControl(2, CHANNEL_TYPE_DAC, CHANNEL_CONTROL_WAVE), str = "TestPulse")
+End
+
+Function TPWaitForAsync_IGNORE(TPStorage, storedTP, timeOut)
+	WAVE TPStorage
+	WAVE storedTP
+	variable timeOut
+
+	variable size1, size2, endTime
+	string errorMsg, empty
+
+	endTime = DateTime + timeOut
+	do
+		ASYNC_ThreadReadOut()
+		size1 = GetNumberFromWaveNote(TPStorage, NOTE_INDEX)
+		size2 = GetNumberFromWaveNote(storedTP, NOTE_INDEX)
+		if(size1 == size2)
+			return 0
+		endif
+	while(DateTime < endTime)
+
+	empty = ""
+	errorMsg = "TimeOut for TP data was reached"
+	CHECK_EQUAL_STR(errorMsg, empty)
+End
+
+
+static Constant TP_WIDTH_EPSILON = 1
+static Constant TP_DYN_PERC_TRESHOLD = 0.2
+static Constant TP_EDGE_EPSILON = 0.2
+static Constant TP_SKIP_NUM_TPS_START = 1
+
+// UTF_TD_GENERATOR HardwareMain#DeviceNameGeneratorMD1
+Function TPDuringDAQTPStoreCheck([str])
+	string str
+
+	STRUCT DAQSettings s
+	InitDAQSettingsFromString(s, "MD1_RA0_I0_L0_BKG_1_RES_1")
+	AcquireData(s, str, preAcquireFunc=TPDuringDAQTPStoreCheck_IGNORE)
+End
+
+Function TPDuringDAQTPStoreCheck_REENTRY([str])
+	string str
+
+	CheckStoredTPs_IGNORE(str, 2)
+End
+
+Function CheckStoredTPs_IGNORE(device, expectedNumTPchannels)
+	string device
+	variable expectedNumTPchannels
+
+   variable i, channel, numTPChan, numStored, numTP, tpLength
+   variable tresh, m, tpWidthInMS
```

Comment:
indentation

Reply:
As this is invisible in Igor, would it maybe convenient to add a warning in fixws.sh for lines that start with a indentation using space for ipf files?

Reply:
@MichaelHuth Here my pre-commit hooks catches that.

```
$ ls .git/hooks/pre-commit
.git/hooks/pre-commit*
```

Reply:
Ok, we found why I got no notification about that.